src/main/scala/io/iohk/praos/util/TransactionsGenerator.scala

```diff
@@ -0,0 +1,48 @@
+package io.iohk.praos.util
+
+import scala.annotation.tailrec
+import scala.util.Random
+import io.iohk.praos.crypto.Key
+import io.iohk.praos.domain.{Stake, StakeDistribution, Transaction, applyTransaction}
+
+object TransactionsGenerator {
+
+  type Stakeholder = (Key, Stake)
```

Comment:
Minor detail: Maybe this type alias should be `private` to differentiate from the `case class StakeHolder`?